Examples/Algorithms/TrackFitting/src/KalmanFitterFunction.cpp

```diff
@@ -67,6 +67,60 @@ struct SimpleReverseFilteringLogic {
   }
 };
 
+template <typename PredictedPars, typename PredictedCov>
+double calculateChi2(const double* fullCalibrated,
```

Comment:
@andiwand I think I copied this from the measurement selector or so, maybe this can be somehow generalized?
But not sure where the best place would be...

Reply:
does this help? https://github.com/acts-project/acts/blob/main/Core/include/Acts/Utilities/TrackHelpers.hpp#L639